fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -1179,10 +1237,10 @@ private FHIRPersistenceNotSupportedException buildNotSupportedException(String m
             }
 
             // Fetch the resource payload if needed and convert to a model object
-            final T resource = readResource(resourceType, resourceDTO, elements);
+            final T resource = convertResourceDTO(resourceDTO, resourceType, elements);
 
             SingleResourceResult<T> result = new SingleResourceResult.Builder<T>()
-                    .success(true)
+                    .success(true) // TODO even if resource is null?
```

Comment:
how to resolve?

Reply:
answer is yes because the REST layer now handles null